uint32 id
float32 dlon_rms
float32 vlon_rms
float32 dlat_rms
float32 vlat_rms
float32 ori_rms
